backend/buoyParser.py

```diff
@@ -0,0 +1,71 @@
+import requests
+from dbClass import Database
```

Comment:
```suggestion
from db_class import Database
```
according to python style guide variables and modules are using `snake_case`, it is important to follow consistent style guide when you work with bunch of people on the project to make it look like only a single person wrote the code. Different languages would have different style guides, you could see some python style guides here:
https://peps.python.org/pep-0008/
https://google.github.io/styleguide/pyguide.html

additionally you may want to setup `linters` (the most common is "flake8") and `auto-formaters` (the most common is "black") as a part of "build" process that would signal what you are doing not according to the style guide or would fix it for you see https://github.com/vintasoftware/python-linters-and-code-analysis for more details